node/src/components/consensus/highway_core/highway/vertex.rs

```diff
@@ -164,6 +164,11 @@ impl<C: Context> WireVote<C> {
     pub(crate) fn round_id(&self) -> Timestamp {
         state::round_id(self.timestamp, self.round_exp)
     }
+
+    /// Returns whether `vhash` is endorsed by this vote.
```

Comment:
But it's not endorsed by this vote, is it?
This vote claims that `vhash` is endorsed, and depends on endorsements for it.

Reply:
But we download all dependencies before validating the vote, so we should have those votes seen as endorsed in our local state.

Reply:
Ah, it's just the comment that's confusing. Sorry.

Reply:
Updated the comment.